modules/user-access/pages/rbac-row-policy/row-policy-privlages-table.adoc

```diff
@@ -0,0 +1,138 @@
+= RBAC: Row Policy Privileges Tables
```

Comment:
Actually it is not row policy only feature. It will affect all existing privileges.
We need to add a link to this page as well.
https://docs.tigergraph.com/tigergraph-server/current/reference/list-of-privileges

Reply:
Thank you Added some limitations and warnings and did some renaming as we discussed on zuilp.